pyspecProcScripts/phasing.py

```diff
@@ -201,7 +201,6 @@ def hermitian_function_test(
     frq_range=(-2.5e3, 2.5e3),  # assumes signal is somewhat on resonance
     aliasing_slop=3,
     band_mask=False,
-    final_range=(100e-5, None),
```

Comment:
we can't do this b/c it completely freaks out of the axis starts w/ a negative number (which we should allow)